components/datepicker/stories/datepicker.mdx

```diff
@@ -59,3 +54,4 @@ Read-only displays the same for standard and quiet variants.
 ## Properties
 
 <ArgTypes />
+../metadata/datepicker.stories../metadata/datepicker.stories../stories/datepicker.stories
```

Comment:
I think this may have been inadvertently added to the file?

Reply:
Yes, I do believe you are right!